apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.js

```diff
@@ -31,6 +31,94 @@ function getTourAssets( key ) {
 	return tourAssets[ key ];
 }
 
+function getTourDescriptions( key, localeSlug = null ) {
+	const tourDescriptions = {
+		welcome: {
+			desktop: __(
+				'Take this short, interactive tour to learn the fundamentals of the WordPress editor.',
+				'full-site-editing'
+			),
+			mobile: null,
+		},
+		allBlocks: {
+			desktop: __(
+				'In the WordPress Editor, paragraphs, images, and videos are all blocks.',
+				'full-site-editing'
+			),
+			mobile: null,
+		},
+		addBlocks: {
+			desktop: __(
+				'Click + to open the inserter. Then click the block you want to add.',
+				'full-site-editing'
+			),
+			mobile:
+				localeSlug === 'en' ||
+				hasTranslation?.( 'Tap + to open the inserter. Then tap the block you want to add.' )
+					? __(
+							'Tap + to open the inserter. Then tap the block you want to add.',
+							'full-site-editing'
+					  )
+					: __(
+							'Click + to open the inserter. Then click the block you want to add.',
+							'full-site-editing'
+					  ),
+		},
+		makeBold: {
+			desktop: __(
+				'Use the toolbar to change the appearance of a selected block. Try making it bold.',
+				'full-site-editing'
+			),
+			mobile: null,
+		},
+		moreOptions: {
+			desktop: __( 'Click the settings icon to see even more options.', 'full-site-editing' ),
+			mobile:
+				localeSlug === 'en' || hasTranslation?.( 'Tap the settings icon to see even more options.' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Click the settings icon to see even more options.' )` *ES Score: 6*